quantum/pointing_device/pointing_device.c

```diff
@@ -450,6 +454,28 @@ report_mouse_t pointing_device_adjust_by_defines_right(report_mouse_t mouse_repo
     return mouse_report;
 }
 
+/**
+ * @brief Handle core combined pointing device tasks
+ *
+ * Takes 2 report_mouse_t structs allowing individual modification of either side and then returns pointing_device_task_combined_kb.
+ *
+ * NOTE: Only available when using both SPLIT_POINTING_ENABLE and POINTING_DEVICE_COMBINED
+ *
+ * @param[in] left_report report_mouse_t
+ * @param[in] right_report report_mouse_t
+ * @return pointing_device_task_combined_kb(left_report, right_report) by default
+ */
+report_mouse_t pointing_device_task_combined(report_mouse_t left_report, report_mouse_t right_report) {
+#    ifdef POINTING_DEVICE_SCROLL_ENABLE
+    if (is_scroll_mode_left()) {
+        left_mouse_report = pointing_device_task_scroll(left_mouse_report);
```

Comment:
this woulndn't build for me unless I change it to 

>     if (is_scroll_mode_left()) {
>         local_mouse_report = pointing_device_task_scroll(left_report);
>     } else {
>         local_mouse_report = pointing_device_task_scroll(right_report);
>     }

Reply:
Ah yes there is a very obvious typo in there thanks for checking this.
will make sure that this builds with all options before updating with the new version as I plan on moving this out of draft status then as well.

should be:
```c
if (is_scroll_mode_left())
    left_report = pointing_device_task_scroll(left_report);
} else {
    right_report = pointing_device_task_scroll(right_report);
}
```